ibm_db2/ibm_db2.php

```diff
@@ -1628,27 +1628,27 @@ function db2_setoption() {}
  * <p>
  * Returns false if no row was retrieved.
  */
-function db2_fetch_object($stmt, $row_number = null) {}
+function db2_fetch_object($stmt, int $row_number = null): stdClass|false {}
```

Comment:
Does the funtion really returns stdClass or just a "some" object? Maybe `object` type hint would be better here?

Reply:
All the `db_*()` functions returning objects are returning instances of `stdClass`. See php/pecl-database-ibm_db2#34.